prac_06/programming_language.py

```diff
@@ -0,0 +1,13 @@
+class ProgrammingLanguage:
+    def __init__(self, name, typing, reflection, year):
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def __str__(self):
+        return '{}, {} Typing, Reflection={}, First appeared in {}'.format(self.name, self.typing, self.reflection,
+                                                                           self.year)
+
+    def is_dynamic(self):
```

Comment:
`def is_dynamic(self) -> bool:`
`    if self.typing == "Dynamic":`
`        return True`
`    else:`
`        return False`

Try this for your is_dynamic function

Reply:
Added the return type however I have kept the main part of the code as it was recommended (or at the very least shown as an example) in previous lectures